src/mod_manager_ui.cpp

```diff
@@ -88,6 +88,11 @@ std::string mod_ui::get_information( MOD_INFORMATION *mod )
              << "</color>: " << str << "\n";
     }
 
+    if( !mod->version.empty() ) {
+        info << "<color_light_blue>" << _( "Mod version" ) << "</color>: " <<
+             ( mod->version.c_str() ) << "\n";
```

Comment:
Can be boiled down to `<< mod->version << std::endl`. Also, formatting (`<< ` aligned vertically).